base/applications/mscutils/eventvwr/lang/cs-CZ.rc

```diff
@@ -216,7 +219,8 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_COPY "Typ události:     %s\r\n\
+    IDS_COPY "\
+Typ:       %s\r\n\
```

Comment:
Why the change? You speak czech?

Reply:
The události == event exceeded the available space here, and required the ':' of the first line to be unaligned to the other ones.